### DISCUSSION TITLE
Email Form to a user after saving

### DISCUSSION THREAD

**Discussion by AzureRepo1** [Budibase/budibase#10685] [Help] [answered]
**Describe the feature request**
hi there, is it possible to send a completed form via email to a user automatically after it is saved?
**Screenshots**

**Comment by ConorWebb96** [comment, marked as answer]
Hey @AzureRepo1,

Yes this can be done with the usage of automations. I'm going to list some steps below to help you with this.

1. Configure the email settings within Budibase, the screenshot below shows an example of how this can be setup.
<img width="100%" alt="Screenshot 2023-05-25 at 08 08 54" src="https://github.com/Budibase/budibase/assets/126772285/2a663322-9f68-4d7c-96b3-7f0b50420305">

2. Create the table's structure with the design area that you want this automation to be configured to, in this example I'm going to create a simple customer table with the fields `First Name`, `Last Name` and email.

3. Next head over to the design tab and add a `Form block` component, set to the newly created customer table.

4. Switch to the automation tab create a new automation

5. Click the add automation button. 

<img width="100%" alt="Screenshot 2023-05-25 at 08 24 08" src="https://github.com/Budibase/budibase/assets/126772285/13722bb8-ce39-4685-8a9d-025088bfbbbe">

6. Select the `Row Created` option and add a relevant name: e.g Send Email
<img width="100%" alt="Screenshot 2023-05-25 at 08 24 22" src="https://github.com/Budibase/budibase/assets/126772285/84aa7895-3264-4f3d-a945-93dd95c62b44">

7. Select the table you wish to run this automation on.
8. Add a step which will be used to send the email.
<img width="100%" alt="Screenshot 2023-05-25 at 08 28 12" src="https://github.com/Budibase/budibase/assets/126772285/33abf3a6-126b-4b8a-86eb-4753e0e074e9">

9. Fill out all the relevant information required, you can use bindings to feed in data from the form.
<img width="100%" alt="Screenshot 2023-05-25 at 08 17 02" src="https://github.com/Budibase/budibase/assets/126772285/bee4c40f-38c2-4612-ad66-7dbd8f2180fb">

10. Publish your app.
11. Go to the screen with your form fill in the details and send, this should run the automation and send the email.

For this demo I'm simply outputting the entire row which is created via the form, you will likely want to template out things such as: First Name, last name etc. I've shown an example of how you could do this with the binding above screenshot.

**Results of automation**
<img width="382" alt="Screenshot 2023-05-25 at 08 14 29" src="https://github.com/Budibase/budibase/assets/126772285/af9e288f-fdbb-409f-9ac2-b963e3769998">

<img width="639" alt="Screenshot 2023-05-25 at 08 15 15" src="https://github.com/Budibase/budibase/assets/126772285/52c03c9e-4f00-4aa4-9e6c-8ddeb6e2bb97">

I hope this helps you with understanding how to do this within Budibase.

**Reply by melohagan** [reply]
As a side note you can also directly submit form content to an automation using the [App action](https://docs.budibase.com/docs/app-action).

Define fields in the trigger for each of your form fields, and then pass them through form the Design section:

![Screenshot 2023-06-14 at 15 30 24](https://github.com/Budibase/budibase/assets/101575380/21d0a780-230a-41d6-b3e4-a6d02975e27f)

![Screenshot 2023-06-14 at 15 31 36](https://github.com/Budibase/budibase/assets/101575380/f35fabf5-51ef-4bb9-8846-208e1a0dd615)
